### DISCUSSION TITLE
Allow to configure name for base accounts (like Expense, Assets, Revenue, …)

### DISCUSSION THREAD

**Discussion by matclab** [ananthakumaran/paisa#64] [Ideas]
You piece of software looks very nice, however I already use ledger with localized account names (Dépense, Revenus, Actifs, Passifs) and as such the UI does not provide reports.

It would be nice to allow to customize the name of the accounts expected by paisa.

**Comment by ananthakumaran** [comment]
I will think about it, but it's too early for this now. Compared to ledger, paisa will be more opinionated because ledger itself doesn't care about whether you are spending money or investing, it mostly cares about whether the posting balances to zero.

I assume this is more of an i18n request, I need to stabilize a lot of other things before I can think about i18n.

**Reply by matclab** [reply]
Yep, I understand your concerns. 

Thank you for your answer, I'll definitively have an eye on paisa!

**Reply by jasperrooduijn** [reply]
Any news on this? I have the same problem with my Dutch  (nl_NL) setup of ledger-cli. Adding some kind of i18n would be great. I would love to help translating to Dutch for you.

**Reply by ananthakumaran** [reply]
Not much progress here, there are 2 independent requests here

1) Make the account name configurable. This requires some restructuring of existing code and I have been focusing on a few other features, I will eventually get to it
2) i18n, this is just low on priority as of now.

**Reply by jasperrooduijn** [reply]
Understood and agreed. For me personally number one is the only real important one here.

**Reply by archtur** [reply]
I have the same issue with german account names. It should be simply configurable like

```yaml
account_names:
    assets:
        - Ausgaben:*
    liabilities:
        - Verbindlichkeiten:*
    income:
        - Einnahmen:*
    expenses:
        - Ausgaben:*
    equity:
        - Eigenkapital:*
```

Each category can be configured with multiple patterns, like described in https://paisa.fyi/reference/config/#accounts.

With this approach you would have a generic solution for all kind of ledger / hleder users. I think this is a common problem with paisa, because in ledger / hledger you are very free to configure all the stuff like you need and hldeger journal files you can also configure the account types completely flexible, see https://hledger.org/1.51/hledger.html?highlight=account%20type#account-types.

**Comment by pashagolub** [comment]
@matclab Just an idea. Can you use `alias` command in your ledger files for this? That way in the beginning you will define localized names and english aliases for them. Not sure it's a good idea. Never tried it myself

**Reply by ananthakumaran** [reply]
One of the users reported that the alias was working with hledger https://github.com/ananthakumaran/paisa/issues/106.
